x-pack/plugins/siem/public/lib/clipboard/clipboard.tsx

```diff
@@ -82,15 +48,23 @@ export class Clipboard extends React.PureComponent<Props, State> {
     }
 
     if (isSuccess) {
-      this.setState(prevState => ({
-        toasts: [...prevState.toasts, getSuccessToast({ titleSummary })],
-      }));
+      dispatchToaster({
+        type: 'addToaster',
+        toast: { toastLifeTimeMs, ...getSuccessToast({ titleSummary }) },
+      });
     }
   };
 
-  private removeToast = (removedToast: Toast): void => {
-    this.setState(prevState => ({
-      toasts: prevState.toasts.filter(toast => toast.id !== removedToast.id),
-    }));
-  };
-}
+  // TODO: 1 error is: Visible, non-interactive elements with click handlers must have at least one keyboard listener
+  // TODO: 2 error is: Elements with the 'button' interactive role must be focusable
+  // TODO: Investigate this error
+  /* eslint-disable */
+  return (
+    <>
+      <div role="button" data-test-subj="clipboard" onClick={onClick}>
```

Comment:
Wait, is this copied from `public/components/timeline/properties/helpers.tsx`?

This copied code looks very unhealthy and odd here and the place it was copied from.

Can we just use a regular `EuiButton` and not copy the debt into this file?

Reply:
If EUI button can't be used because of styling, I would use a `<button` and testing it seems to work out ok:

```ts
export const Clipboard = ({ children, content, onCopy, titleSummary, toastLifeTimeMs }: Props) => {
  const dispatchToaster = useStateToaster()[1];
  const onClick = (event: { preventDefault: () => void; stopPropagation: () => void }) => {
    event.preventDefault();
    event.stopPropagation();

    const isSuccess = copy(`${content}`, { debug: true });

    if (onCopy != null) {
      onCopy({ content, isSuccess });
    }

    if (isSuccess) {
      dispatchToaster({
        type: 'addToaster',
        toast: { toastLifeTimeMs, ...getSuccessToast({ titleSummary }) },
      });
    }
  };

  return (
    <>
      <button type="button" data-test-subj="clipboard" onClick={onClick}>
        {children}
      </button>
    </>
  );
};
```